Sources/Workspace/Workspace.swift

```diff
@@ -734,19 +734,9 @@ extension Workspace {
         diagnostics: DiagnosticsEngine
     ) -> [Manifest] {
 
-        let lock = Lock()
-        let sync = DispatchGroup()
-        var rootManifests = [Manifest]()
-        packages.forEach { package in
-            self.queue.async(group: sync) {
-                if let manifest = self.loadManifest(packagePath: package, url: package.pathString, packageKind: .root, diagnostics: diagnostics) {
-                    lock.withLock {
-                        rootManifests.append(manifest)
-                    }
-                }
-            }
-        }
-        sync.wait()
+        let rootManifests = packages.compactMap({ package -> Manifest? in
```

Comment:
I could see this being helpful once large projects are regularly composed of lots of packages, but for now loading these in sequence seems like the right approach.  That would also have any diagnostics show up in the same order every time (assuming the input array is stable, which I think it is), which is nice.

Reply:
I realized late that these are the root manifest, which is usually not the issue. I moved the parallelization to the top level dependencies

Reply:
Right, agreed.  For command-line SwiftPM it's just the one.  For something like an Xcode workspace, there could potentially be lots of root packages.  But it's likely to be a while before that's a common situation, so I think doing it just for dependencies is the right thing for now.